pandas/core/frame.py

```diff
@@ -5053,8 +5053,9 @@ def infer(x):
 
     def append(self, other, ignore_index=False, verify_integrity=False):
         """
-        Append rows of `other` to the end of this frame, returning a new
-        object. Columns not in this frame are added as new columns.
+        Append rows of `other` to the end of `caller`, returning a new object.
```

Comment:
'caller' is not needed to quote (it's not an argument). 

Also given this is the docstring specific to DataFrame (not shared with Series), I think we can be more specific. So maybe "calling DataFrame" ? (or is it then too long ?)